OracleHTTPServer/dockerfiles/12.2.1.2.0/Dockerfile

```diff
@@ -4,60 +4,63 @@
 #
 # ORACLE HTTP Server DOCKERFILES PROJECT
 # -----------------------------------------
-# This is the DockerFile for Oracle HTTP Server 12.2.1
+# This is the DockerFile for Oracle HTTP Server 12.2.1.2.0
 #
 # IMPORTANT
 # -------------------------------------------
-# The resulting image of this Dockerfile would have Oracle Linux 7 and Oracle HTTP Server 1221 installed in standalone mode.
-# No domain will be configured in this image
-#
-# For Creating a OHSdomain out of above image
-# You can go into 'samples/1221-OHS-domain' and build image using DockerFile provided in it
+# The resulting image of this Dockerfile would have Oracle Linux 7.3 and Oracle HTTP Server 12.2.1.2.0 installed in standalone mode.
+# A standalone domain will be configured in this image
 #
 # REQUIRED FILES TO BUILD THIS IMAGE
 # ----------------------------------
-# (1) fmw_12.2.1.0.0_ohs_linux64_Disk1_1of1.zip
+# (1) fmw_12.2.1.2.0_ohs_linux64_Disk1_1of1.zip
 #     Download from http://www.oracle.com/technetwork/middleware/webtier/downloads/index-jsp-156711.html
 #
-# (2) server-jre-8uXX-linux-x64.tar.gz
-#     Download from http://www.oracle.com/technetwork/java/javase/downloads/server-jre8-downloads-2133154.html
-#
 # HOW TO BUILD THIS IMAGE
 # -----------------------
 # Put all downloaded files in the same directory as this Dockerfile
 # As root user run following command manually:
 #
-#     $ sh buildDockerImage.sh -v 12.2.1
+#     $ sh buildDockerImage.sh -v 12.2.1.2.0
 #
 # Alternatively you can also run
 # 
 #     $ docker build --force-rm=true --no-cache=true --rm=true -t oracle/$IMAGE_NAME:$VERSION-$INSTALL_TYPE .
 #
 # For example
 # 
-#     $ docker build --force-rm=true --no-cache=true --rm=true -t oracle/ohs:12.2.1-sa .
+#     $ docker build --force-rm=true --no-cache=true --rm=true -t oracle/ohs:12.2.1.2.0-sa .
 #
-# Pull Oracle Linux 7 base image
+# Pull serverjre:8 base image
 # -----------------------------------
-FROM oracle/serverjre:8
+FROM container-registry.oracle.com/java/serverjre:8
```

Comment:
Do not specify the container-registry URL here. Customers can build `java/serverjre:8` locally or could use the Docker Store version too.